tests/test1.cpp

```diff
@@ -37,6 +37,13 @@ TEST_CASE("Parse Complex") {
         }"));
 }
 
+TEST_CASE("Parse Multiple in any order") {
+    REQUIRE(parse_helper("{\
+        Input{\"amool\" , \"bmpp\", \"cmqq\"}  \
+        Button{\"amool\" , \"bmpp\"}\
+        }"));
+}
+
```

Comment:
https://github.com/SAtacker/quick-ftxui/pull/5#discussion_r1096808138
Added test case for checking grammer, using 'Input' component